tests/10-rdfc-create.js

```diff
@@ -78,6 +96,13 @@ describe('eddsa-rdfc-2022 (create)', function() {
           ).should.equal(true, 'Expected at least one proof to have "type" ' +
             'property value "Multikey".');
         });
+        /**
+         * @see {@link https://www.w3.org/TR/vc-di-eddsa/#dataintegrityproof}
+         *
+         * "The proofPurpose property of the proof MUST be a string,
+         * and MUST match the verification relationship expressed
+         * by the verification method controller."
+         */
         it('The "proof.proofPurpose" field MUST match the verification ' +
```

Comment:
when will you update the test title? We should be updating the test titles.

Reply:
Ill update all the test titles as a separate PR if that works for you